lib/events.js

```diff
@@ -47,7 +47,7 @@ EventEmitter.prototype.setMaxListeners = function setMaxListeners(n) {
 };
 
 function $getMaxListeners(that) {
-  if (util.isUndefined(that._maxListeners))
+  if (that._maxListeners === void 0)
```

Comment:
`undefined` is not re-definable in ES5. Is there a good reason to use `void 0` over it?